cpp/dolfinx/mesh/utils.h

```diff
@@ -108,5 +108,24 @@ Eigen::Array<std::int32_t, Eigen::Dynamic, 1> locate_entities_boundary(
         const Eigen::Ref<const Eigen::Array<double, 3, Eigen::Dynamic,
                                             Eigen::RowMajor>>&)>& marker);
 
+/// Compute the geometry indices of given entities from the mesh geometry
+/// @param[in] mesh Mesh
+/// @param[in] dim Topological dimension of the entities of interest
+/// @param[in] entity_list List of entities to obtain vertex geometry
+/// @param[in] orient If true, in 3D, reorients facets to have consistent normal
+/// direction
+/// @return Vertex indices in the geometry dofmap of the mesh for each entity
+Eigen::Array<std::int32_t, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>
+entities_to_geometry(
+    const mesh::Mesh& mesh, const int dim,
+    const Eigen::Array<std::int32_t, Eigen::Dynamic, 1>& entity_list,
+    bool orient);
+
+/// Compute exterior facet indices only
```

Comment:
`/// Compute the indices (local) of all exterior facets. An exterior facet (co-dimension 1) is one that is connected globally to only one cell of co-dimension 0).`